src/app/notes/notes.service.ts

```diff
@@ -19,7 +20,17 @@ export class NotesService {
    * @returns The NoteList as observable
    */
   getNotes(): Observable<NoteList> {
-    this.logger.debug(`Gathering notes from ${environment.notesAsset}`);
-    return this.http.get<NoteList>(environment.notesAsset).pipe();
+    this.logger.debug(`Gathering notes from ${environment.notesAssets}`);
+    return forkJoin(
+      this.http.get<NoteList>(environment.notesAssets[0]),
+      this.http.get<NoteList>(environment.notesAssets[1]),
+    ).pipe(
+      map((notes: [NoteList, NoteList]) => {
+        for (const key of Object.keys(notes[0])) {
```

Comment:
@jeefy Usually I would like to do something like this here:
```typescript
return new Map([...notes[0], ...notes[1]]);
```

Unfortunately this does not work since it seems we're loosing our type `Map<...>` here. If I try to access any property of the `Map`, it results in a runtime type error (:astonished:) like this:
```
TypeError: notes[0].keys is not a function
```

Why don't we get the correct type passed over here? What do you think?

Reply:
@Drogglbecher, do you have any clue? :point_up:

Reply:
`notes[0]` is at least not the correct way to access the member of a `Map` but an associative array, try `notes.get(0).keys()` :)